sdks/python/apache_beam/examples/inference/vertex_ai_image_classification.py

```diff
@@ -0,0 +1,158 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+""" A sample pipeline using the RunInference API to classify images of flowers.
+This pipeline reads an already-processes representation of an image of
+sunflowers and sends it to a deployed Vertex AI model endpoint, then
+returns the predictions from the classifier model. The model and image
+are from the Hello Image Data Vertex AI tutorial (see
+https://cloud.google.com/vertex-ai/docs/tutorials/image-recognition-custom
+for more information.)
+"""
```

Comment:
Note: we'll eventually want a notebook that demonstrates all of this as well (not in the scope of this PR)

Reply:
+1